### ISSUE TITLE
cni with none driver on  latest kubernetes version not working

### ISSUE THREAD

**Issue by subhamkrai** [manusa/actions-setup-minikube#65]
this what we have in config 
```
- name: setup minikube
      uses: manusa/actions-setup-minikube@v2.6.0
      with:
        minikube version: "v1.26.0"
        kubernetes version: "v1.24.1"
        start args: --memory 6g --cpus=2 --addons ingress --cni=flanne
```
logs
```
X Exiting due to RUNTIME_ENABLE: sudo systemctl restart cri-docker: exit status 1
stdout:
stderr:
Job for cri-docker.service failed because the control process exited with error code.
See "systemctl status cri-docker.service" and "journalctl -xe" for details.
....
Error: Command failed: sudo -E /home/runner/work/_temp/minikube start --vm-driver=none  --kubernetes-version v1.24.1 --memory 6g --cpus=2 --addons ingress --cni=flannel
    at checkExecSyncError (child_process.js:635:11)
Error: Command failed: sudo -E /home/runner/work/_temp/minikube start --vm-driver=none  --kubernetes-version v1.24.1 --memory 6g --cpus=2 --addons ingress --cni=flannel
    ....l
}
Error: Command failed: sudo -E /home/runner/work/_temp/minikube start --vm-driver=none  --kubernetes-version v1.24.1 --memory 6g --cpus=2 --addons ingress --cni=flannel
```

link of the action https://github.com/rook/rook/runs/7246482502?check_suite_focus=true#step:3:164

**Comment by subhamkrai** [comment]
@manusa any idea?

this is impacting our upgrade to the latest k8s version

**Comment by manusa** [comment]
Should be addressed by #95 

Latest test scenarios pass.

Just released v2.8.0 https://github.com/marketplace/actions/setup-minikube-kubernetes-cluster?version=v2.8.0 with the changes
